Sources/RxMoya/RxMoyaProvider.swift

```diff
@@ -6,22 +6,35 @@ import Moya
 
 /// Subclass of MoyaProvider that returns Observable instances when requests are made. Much better than using completion closures.
 open class RxMoyaProvider<Target>: MoyaProvider<Target> where Target: TargetType {
+    
+    /// Propagated to Alamofire as callback queue. If nil - main queue will be used.
+    fileprivate let queue: DispatchQueue?
+    
     /// Initializes a reactive provider.
-    override public init(endpointClosure: @escaping EndpointClosure = MoyaProvider.defaultEndpointMapping,
+    public init(endpointClosure: @escaping EndpointClosure = MoyaProvider.defaultEndpointMapping,
```

Comment:
why is override no longer needed here? is it because you added the `queue` param? If so, should we not also add the `queue` param to the regular callback `MoyaProvider`?